### PR TITLE
[IMP] crm: void helper wording

### PR DESCRIPTION
**Pull request by maan-odoo**
In the current version, the wording of the helper in CRM does not have a
hyperlink to redirect to the sales team if the user does not have a member
of any sales team.

This commit Improve the wording of the helper and also add the hyperlink to
redirect the sales team if the user does not have a member of any sales team
and has enough access rights.

task-2846401

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm/models/crm_team.py [hunk 1]
```diff
@@ -608,13 +608,15 @@ def _action_update_to_pipeline(self, action):
             user_team_id = self.search([('id', '=', user_team_id)], limit=1).id
         else:
             user_team_id = self.search([], limit=1).id
-            action['help'] = _("""<p class='o_view_nocontent_smiling_face'>Add new opportunities</p><p>
-    Looks like you are not a member of a Sales Team. You should add yourself
-    as a member of one of the Sales Team.
-</p>""")
+            action['help'] = _("<p class='o_view_nocontent_smiling_face'>Create an Opportunity</p>")
```
Comment (tde-banana-odoo) [inline addons/crm/models/crm_team.py:611]: While modifying this code, I think we should take the opportunity to extract sentences to translate independently from the html itself.

Comment (maan-odoo) [reply addons/crm/models/crm_team.py:611]: Indeed we can do that for a single sentence. But in the changes below the condition, we need to inject parts of the sentence between html tags (bold, anchor...). And if we translate parts separately, aren't they more likely to be broken grammatically?

Comment (tde-banana-odoo) [reply addons/crm/models/crm_team.py:611]: We could keep the bold tags (not that complicated) and already remove extra <p> for example.

Comment (tde-banana-odoo) [reply addons/crm/models/crm_team.py:611]: In short: ok, we can just add the p tags as extra, the remaining is probably better kept as it is.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**